tests/test_python.py

```diff
@@ -106,13 +115,27 @@ def test_tox(auto_container):
     "container_per_test", CONTAINER_IMAGES_T1, indirect=["container_per_test"]
 )
 @pytest.mark.parametrize("hmodule, port, retry", [("http.server", port1, 10)])
-def test_python_webserver_1(container_per_test, hmodule, port, retry):
+def test_python_webserver_1(
+    container_per_test, container_runtime, hmodule, port, retry
+):
     """Test that the python webserver is able to open a given port"""
 
     portstatus = False
-
+    podman_version_x = None
     t = 0
 
+    if container_runtime.runner_binary == "podman":
+        podman_version = container_version(container_runtime).split(".")
+        assert podman_version[0].isnumeric()
+
+        # container version: old versions have issues with background processes
+        if int(podman_version[0]) < 2:
+            pytest.xfail(
+                "server port checks not compatible with old podman versions 1"
+            )
```

Comment:
Please skip the whole test, there's really no need to run it if we know that it is broken